test/common/http/codec_impl_fuzz_test.cc

```diff
@@ -80,9 +80,11 @@ class HttpStream : public LinkedObject<HttpStream> {
   // course, it's the codecs must be robust to wire-level violations. We
   // explore these violations via MutateAction and SwapAction at the connection
   // buffer level.
-  enum class StreamState { PendingHeaders, PendingDataOrTrailers, Closed };
+  enum class StreamState : int { PendingHeaders, PendingDataOrTrailers, Closed };
```

Comment:
Why the int?

Reply:
To allow it to be converted to an int in the debug log statements.

Reply:
We do this elsewhere without doing this using `enumToInt` for example, but nbd.